frontend/src/pages/admin/NotificationsHub.vue

```diff
@@ -1,84 +1,82 @@
+                        <template #input><textarea v-model="form.message" class="w-full max-h-80 min-h-40" rows="4" /></template>
+                    </FormRow>
+                    <FormRow v-model="form.url" type="input" :placeholder="urlPlaceholder" class="mb-5" data-el="notification-external-url">
+                        URL Link
+                        <template #description>Provide an url where users will be redirected when they click on the notification.</template>
+                    </FormRow>
+                </section>
+                <section>
+                    <FormHeading>Audience</FormHeading>
+                    <div class="ff-description mb-2 space-y-1">Select the audience of your announcement.</div>
+                    <FormHeading class="mt-4">User Roles:</FormHeading>
+                    <div class="grid gap-1 grid-cols-2 items-middle">
+                        <label
+                            v-for="(role, $key) in roleIds"
+                            :key="$key"
+                            class="ff-checkbox text-sm"
+                            :data-el="`audience-role-${role}`"
+                            @keydown.space.prevent="toggleRole(role)"
+                        >
+                            <span ref="input" class="checkbox" :checked="form.roles.includes(role)" tabindex="0" @keydown.space.prevent />
+                            <input v-model="form.roles" type="checkbox" :value="role" @keydown.space.prevent>
+                            {{ role }}
+                        </label>
+                    </div>
+                    <FormHeading class="mt-4">Team Types:</FormHeading>
+                    <div class="grid gap-1 grid-cols-2 items-middle">
+                        <label
+                            v-for="teamType in teamTypes"
+                            :key="teamType.id"
+                            class="ff-checkbox text-sm"
+                            :class="!teamType.active ? ['inactive-team'] : []"
+                            :data-el="`audience-teamType-${teamType.id}`"
+                            @keydown.space.prevent="toggleTeamType(teamType.id)"
+                        >
+                            <span ref="input" class="checkbox" :checked="form.teamTypes.includes(teamType.id)" tabindex="0" @keydown.space.prevent />
+                            <input v-model="form.teamTypes" type="checkbox" :value="teamType.id" @keydown.space.prevent>
+                            {{ teamType.name }}
+                        </label>
+                    </div>
+                    <template v-if="features.billing">
+                        <FormHeading class="mt-4">Billing State:</FormHeading>
                         <div class="grid gap-1 grid-cols-2 items-middle">
                             <label
-                                v-for="(role, $key) in roleIds"
+                                v-for="(billingState, $key) in billingStates"
```

Comment:
These changes are misleading in the diff - given we lost loads of code, the alignment means the diff thinks these changed from `roleIds` to `billingStates`, but it is just a shift of code up the file